integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

```diff
@@ -143,7 +143,7 @@ public static boolean scaleCluster(String clusterName, String namespace, int rep
   public static DomainResource addClusterToDomain(String clusterName, String namespace,
                                                   DomainResource domain, int replicas) {
     ClusterList clusters = listClusterCustomResources(namespace);
-    if (clusters.getItems().stream().anyMatch(cluster -> cluster.getClusterName().equals(clusterName))) {
+    if (clusters.getItems().stream().anyMatch(cluster -> cluster.getClusterResourceName().equals(clusterName))) {
```

Comment:
Good catch :-)
Can you please change the method parameter from clusterName to clusterResourceName also to match with what is being checked?

Reply:
done